MCMC/simulation.py

```diff
@@ -197,11 +202,37 @@ def visualize(self, frame_number=0, save_path=None):
             fig_name = "system_frame_" + str(frame_number)
             plt.savefig(os.path.join(save_path, fig_name))
         return plt
-
-    def trajectory(self, save_path=""):
+    
+    def save_system(self, frame=-1):
         """
-        Find a way to play the trajectory from the system history.
-        Note: matplotlib.animation might be helpful(https://matplotlib.org/stable/api/animation_api.html)
-        :param save_path: Path to save the trajectory.
+        Save a single snapshot of system_history to a .txt file.
+        :param frame: Index number of system_history
         """
+        system_array = self.system_history[frame]
+        np.savetxt(fname="system.txt", X=system_array)
+
+    def _update_trajectory(self):
+        with gsd.hoomd.open("trajectory.gsd", "wb") as traj:
```

Comment:
I think `wb` mode overwrites the existing file. [Append mode](https://gsd.readthedocs.io/en/v2.7.0/fl-examples.html#write-a-file-in-append-mode) in gsd is `ab`.